scripts/transformer.sh

```diff
@@ -1,15 +1,23 @@
 python3 ../train_model.py \
     --activation-dropout 0.1 \
-    --dropout 0.1 \
+    --batch-size 128 \
     --checkpoints \
-    --embed-dim 10 \
-    --ffn-dim 10 \
+    --dataset-feature text \
+    --dataset-name c4 \
+    --dataset-subset en \
+    --device cuda \
+    --dropout 0.1 \
+    --embed-dim 128 \
+    --epochs 10 \
+    --ffn-dim 1024 \
     --fsdp \
-    --layers 2 \
-    --lr 0.0001 \
+    --layers 6 \
+    --lr 0.001 \
     --model transformer \
-    --heads 1 \
-    --seq-len 30 \
-    --value-embed-dim 10 \
-    --vocab-size 1000 \
+    --heads 8 \
     --rand-seed 42 \
+    --seq-len 128 \
+    --splits 0.7 0.2 0.1 \
+    --val-freq 3 \
```

Comment:
If you can add 
-t BPT \
that would be good. It will make no difference but make it explicit for testing.